internal/utils/docker.go

```diff
@@ -180,7 +180,7 @@ func GetRegistryAuth() string {
 }
 
 // Defaults to Supabase public ECR for faster image pull
-const defaultRegistry = "public.ecr.aws"
+const defaultRegistry = "docker.io"
```

Comment:
revert this when images are mirrored https://github.com/Logflare/logflare/pull/1418